components/telemetry-operator/controller/tracepipeline/resources.go

```diff
@@ -108,6 +109,7 @@ func getOutputType(output v1alpha1.TracePipelineOutput) string {
 
 func makeExporterConfig(output v1alpha1.TracePipelineOutput) map[string]any {
 	outputType := getOutputType(output)
+	isInsecure := len(strings.TrimSpace(output.Otlp.Endpoint.Value)) > 0 && !strings.HasPrefix(output.Otlp.Endpoint.Value, "https://")
```

Comment:
Let's reverse the condition. TLS should be the default and only if the `http://` prefix is set explicitly, we set the insecure flag. Please also add a unittest.

Reply:
sure, but we want to make https explicit, in case of no scheme provided should be treated as http